leakcanary-android/src/main/java/com/squareup/leakcanary/FragmentRefWatcher.java

```diff
@@ -0,0 +1,51 @@
+package com.squareup.leakcanary;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.support.v4.app.FragmentActivity;
+import android.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentManager.FragmentLifecycleCallbacks;
+
+@TargetApi(26)
+class FragmentRefWatcher {
+
+  public static void install(Activity activity, RefWatcher refWatcher) {
+    new FragmentRefWatcher(refWatcher).watch(activity);
+  }
+
+  private final RefWatcher refWatcher;
+
+  @TargetApi(26)
+  public FragmentRefWatcher(RefWatcher refWatcher) {
+    this.refWatcher = refWatcher;
+  }
+
+  void watch(Activity activity) {
```

Comment:
style consistency nit: some of the visibility in the class is public and here it is package-private. maybe prefer package-private on the other places?

Reply:
Done